reviewrot/__init__.py

```diff
@@ -12,5 +15,9 @@ def get_git_service(git):
     """
     if git == "github":
         return GithubService()
+    elif git == "gitlab":
```

Comment:
Well yeah. Actually I had all of these in a single branch 'dev' earlier but alex preferred to review them in separate branches and so I was trying to break things into multiple branches. Must have left this while branching off.